### PR TITLE
added prettier dependency and script for react code.

### PR DESCRIPTION
**Pull request by sinh343**
I dont know much of anything about symphony so don't know if this can be set up to hook to on-save or something within the encore dev command. For the time being a "prettier" script has been added for developer convenience, `targeting app/Resources/**/*.jsx`

<a href="https://gitpod.io/#https://github.com/AdventureLookup/AdventureLookup/pull/327"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "dev-server-guest": "encore dev-server --hot --host 0.0.0.0 --port 8001",
     "dev-server-host": "encore dev-server --hot --port 8002",
     "dev-server-gitpod": "encore dev-server --hot --port 8002 --public $(gp url 8002) --disable-host-check",
+    "prettier": "prettier --write app/Resources/**/*.jsx",
```
Comment (cmfcmf) [inline package.json:8]: Please change also include JS files. The JS files have not been run through prettier before -> commit the prettified versions as well. In addition, I would like to have a second `lint` command to check whether the files are formatted correctly and rename the formatting command to `format`:
```suggestion
    "format": "prettier --write app/Resources/**/*.{js,jsx}",
    "lint": "prettier --check app/Resources/**/*.{js,jsx}",
```

Comment (sinh343) [reply package.json]: I like this change a lot more, didnt know about the commit hook, added those lines as requested https://github.com/AdventureLookup/AdventureLookup/pull/327/commits/86c82016f088f28132e069f57517e8bab234e29d

File: .travis.yml [hunk 2]
```diff
@@ -72,6 +72,7 @@ script:
       php bin/console lint:twig app
```
Comment (cmfcmf) [inline .travis.yml:72]:
```suggestion
      php bin/console lint:twig app
```

Comment (cmfcmf) [inline .travis.yml:72]:
```suggestion
      php bin/console lint:twig app
```

**Comment by cmfcmf** [comment]
CI is failing because the npm dependencies aren't installed when linting. You should be able to fix that by editing
https://github.com/sinh343/AdventureLookup/blob/86c82016f088f28132e069f57517e8bab234e29d/.travis.yml#L51
to read 
`if [ ${TESTSUITE} == "functional" ] || [ "$TESTSUITE" == "browser" ] || [ "$TESTSUITE" == "lint" ]; then`

**Comment by cmfcmf** [comment]
This PR looks almost ready now - the only thing that's missing is running `npm run format` and committing the formatted `*.js` files. Is that something you could do, @sinh343?

**Pull request merged**